ember-cli-build.js

```diff
@@ -13,7 +17,11 @@ module.exports = function(defaults) {
     },
   };
 
-  if (defaults.project.findAddonByName('ember-native-dom-event-dispatcher')) {
+  if (
+    (project.findAddonByName('ember-native-dom-event-dispatcher') ||
+      emberChecker.isAbove('3.0.0')) &&
+    project.findAddonByName('ember-fetch')
```

Comment:
we're not using `ember-fetch` anywhere though 🤔

Reply:
We do:
- https://github.com/emberjs/ember-test-helpers/blob/master/config/ember-try.js#L119-L124
- https://github.com/emberjs/ember-test-helpers/blob/master/tests/helpers/ajax.js#L16

Reply:
🤦‍♂️  nevermind...